### ISSUE TITLE
Tensorboard Graph

### ISSUE THREAD

**Issue by rezafd23** [fizyr/keras-retinanet#1030]
how to show tensorboard graph like #1003 issue during training? i can't show it like that
thanks

**Comment by hgaiser** [comment]
You can use [`--tensorboard-dir`](https://github.com/fizyr/keras-retinanet/blob/master/keras_retinanet/bin/train.py#L404) to log loss values and then use [`tensorboard`](https://www.tensorflow.org/guide/summaries_and_tensorboard) to visualize them.

**Comment by yustinaivanova** [comment]
@hgaiser Can you please give exact commands how to do it?
